### PR TITLE
CS (Czech) locale

### PR DESCRIPTION
**Pull request by CMIT-RomanP**
This translation was created by myself. I am native speaker.

### CODE REVIEW

**Comment by rparet** [comment]
Great. If we can get +1 CS speaker to review and comment we can merge.

File: app/locales/languages.js [hunk 1]
```diff
@@ -72,6 +73,11 @@ i18next.init({
       translation: {
         label: nllabels,
       },
+    },
+	cs: {
+      translation: {
```
Comment (amasurkar-verimatrix) [inline app/locales/languages.js]: Fix indentations please. Thank you

File: app/locales/languages.js [hunk 2]
```diff
@@ -72,6 +73,11 @@ i18next.init({
       translation: {
         label: nllabels,
       },
+    },
+	cs: {
```
Comment (alpita-masurkar) [inline app/locales/languages.js]: Can you fix indentation for this block?

Comment (CMIT-RomanP) [reply app/locales/languages.js]: Indentation fixed.

**Comment by alpita-masurkar** [comment]
Hi, @CMIT-RomanP , thank you for translating this to Czech! This is amazing!
But there's a couple things that still need to be done before we merge:
1. There have been a few new things added to the files now. Can you have a look and add them too? For example, there is a notification.json file; the line in the overlap.json file has changed:
"overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset",
and the show overlap line too:
"show_overlap": "Click to view the public dataset",

2. The merge conflicts need to be resolved

Can you have a look at these before we merge?

Thank you (also, are you on Slack, we can also sync up there if you're on the group)

**Comment by alpita-masurkar** [comment]
Hi @CMIT-RomanP, can you also resolve the merge conflicts quickly? They should be in the languages file if I'm not mistaken

File: app/locales/cs/overlap.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "Kontrola překrytí",
+  "overlap_para_1": "Zelené kruhy jsou shluky vašich míst, kliknutím je zvětšíte. Červené značky jsou vaše zaznamenané pozice.\n\nStisknutím tlačítka stáhnete veřejná data o potvrzených případech, které jsou znázorněny jako slabé fialové kruhy.",
```
Comment (alpita-masurkar) [inline app/locales/cs/overlap.json]: Does this line translate to:
"overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset",?
I think it is still the line from before update. Can you update this translation?

**Comment by alpita-masurkar** [comment]
The translation is good and done. If the merge conflicts are fixed, this branch can be merged.

**Pull request merged**